patterns.py

```diff
@@ -62,4 +62,7 @@ def railgun(light):
             light.neopixel[pixel] = light.primary
         else:
             light.neopixel[pixel] = (0, 0, 0)
-    light.neopixel.show()
\ No newline at end of file
+    light.neopixel.show()
+    
+    if loopcount - light.pattern_starting_loop > light.PIXEL_COUNT:
```

Comment:
❓ I'm a little confused about what this logic does, could you help me understand? In theory the mod operator should loop nicely already?